### ISSUE TITLE
Somehow the version 0.15.2 did not publish to Open VSX

### ISSUE THREAD

**Issue by lukaszsamson** [elixir-lsp/vscode-elixir-ls#352]
It looks like the newest version available is 0.15.1:

```
asd@fwk ~]$ codium --install-extension elixir-lsp.elixir-ls
Installing extensions...
Installing extension 'elixir-lsp.elixir-ls'...
(node:2209612) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `codium --trace-deprecation ...` to show where the warning was created)
Extension 'elixir-lsp.elixir-ls' v0.15.1 was successfully installed.
[asd@fwk ~]$ codium --list-extensions --show-versions | grep elixir
elixir-lsp.elixir-ls@0.15.1
[asd@fwk ~]$ codium --uninstall-extension elixir-lsp.elixir-ls
Uninstalling elixir-lsp.elixir-ls...
Extension 'elixir-lsp.elixir-ls' was successfully uninstalled!
[asd@fwk ~]$ codium --install-extension elixir-lsp.elixir-ls@0.15.2
Installing extensions...
Extension 'elixir-lsp.elixir-ls@0.15.2' not found.
Make sure you use the full extension ID, including the publisher, e.g.: ms-dotnettools.csharp
Failed Installing Extensions: elixir-lsp.elixir-ls
```

So I probably have to install it like described in the [installation section](https://github.com/elixir-lsp/vscode-elixir-ls/tree/master#installation) of the README because it didn't get released yet?

Sorry for the turmoil but because of the vscode extension version 0.15.1 I thought that I'm installing ElixirLS 0.15.1 all the time.

_Originally posted by @adonig in https://github.com/elixir-lsp/elixir-ls/issues/886#issuecomment-1623409546_

0.15.2 is in
https://marketplace.visualstudio.com/items?itemName=JakeBecker.elixir-ls

but not in
https://open-vsx.org/extension/elixir-lsp/elixir-ls

The publish job succeeded
https://github.com/elixir-lsp/vscode-elixir-ls/actions/runs/5414478389/jobs/9841595057

**Comment by adonig** [comment]
This looks like it could be an [openvsx](https://github.com/eclipse/openvsx) issue and from the recent activity it could be that @amvanbaren 👋 might know what to do in that case.

**Comment by lukaszsamson** [comment]
Published once again as 0.15.3 and this time it worked
